### PR TITLE
Revert PR 5592

### PR DESCRIPTION
**Pull request by driesvints**
This PR reverts https://github.com/sebastianbergmann/phpunit/pull/5592 as requested here: https://github.com/sebastianbergmann/phpunit/pull/5592#issuecomment-1873383731

### CODE REVIEW

File: ChangeLog-10.5.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changes in PHPUnit 10.5
 
-All notable changes of the PHPUnit 10.5 release series are documented in this file using the [Keep a CHANGELOG](https://keepachangelog.com/) principles.
+All notable changes of the PHPUnit 10.5 release series are documented in this file using the [Keep a CHANGELOG](https://keepachangelog.com/) principles
```
Comment (sebastianbergmann) [inline ChangeLog-10.5.md:3]: This is unrelated to the issue at hand.

Comment (driesvints) [reply ChangeLog-10.5.md]: Heya yeah sorry. I needed a way to trigger CI on my fork. I reverted this.

**Comment by sebastianbergmann** [comment]
Once you're done, please squash all commits into a single commit that has a sensible commit message. Thanks!

**Comment by driesvints** [comment]
@sebastianbergmann all done.

File: src/Framework/TestRunner.php [hunk 2]
```diff
@@ -83,8 +82,6 @@ public function run(TestCase $test): void
         $risky      = false;
         $skipped    = false;
```
Comment (Matheus-Martinez) [inline src/Framework/TestRunner.php:83]: teste

**Comment by sebastianbergmann** [comment]
Thank you for your contribution. I appreciate the time you invested in preparing this pull request. However, I have decided not to merge it as I agree with @mvorisek: #5592 fixed a breaking change that was introduced PHPUnit 10.0.0 (compared to PHPUnit 8 and PHPUnit 9) that was unfortunately not caught before.

**Comment by driesvints** [comment]
@sebastianbergmann no worries
